commands/addons/docs.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+let cli     = require('heroku-cli-util');
+let co      = require('co');
+let resolve = require('../../lib/resolve');
+
+function* run (context, heroku) {
+  let id = context.args.addon.split(':')[0];
+  let addon = yield heroku.get(`/addon-services/${encodeURIComponent(id)}`).catch(() => null);
+  if (!addon) addon = (yield resolve.addon(heroku, context.app, id)).addon_service;
+  let url = `https://devcenter.heroku.com/articles/${addon.name}`;
```

Comment:
This threw me for a moment because the `addon` var is actually an `addon_service`. Maybe rename the var or access the `addon_service.name` in the URL?